OracleWebLogic/dockerfiles/14.1.1.0/Checksum.developer

```diff
@@ -0,0 +1,6 @@
+# Download WebLogic Server Quick Installer 12.2.1.4
```

Comment:
Will need to update these to describe how to access 14.1.1.0 beta program.

Reply:
Yes